packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-requirements-array.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+  isSecuritySchemeElement,
+  isComponentsElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement, toValue, isObjectElement, Element } from '@swagger-api/apidom-core';
+
+import type { Toolbox } from '../toolbox';
+
+type SecurityRequirementsArrayPluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securityRequirementsArrayRefractorPlugin =
+  ({ annotations }: SecurityRequirementsArrayPluginOptions) =>
+  (toolbox: Toolbox) => {
+    const relevantSecuritySchemes: SecuritySchemeElement[] = [];
+
+    const createAnnotation = <T extends Element>(element: T) =>
+      toolbox.createAnnotation.fromElement(
+        element,
+        'List of roles for Security Requirement Object  is not supported in OpenAPI 3.0.3 for Security Scheme Object types other than "oauth2" and "openIdConnect". As a result, all Security Requirement Objects with Security Scheme Object type other than "oauth2" and "openIdConnect" have been set to an empty array.',
+        { classes: ['error'] },
```

Comment:
This is more a warning than an error. 

```suggestion
        { classes: ['warning'] },
```